tests/spread/general/entrypoint-service/task.yaml

```diff
@@ -0,0 +1,15 @@
+summary: container entrypoint-service test
+
+execute: |
+  run_rockcraft pack
+
+  test -f entrypoint-service-test_latest_amd64.rock
+  test -f entrypoint-service-test_latest_amd64v2.rock
+
+  # test container execution
+  sudo /snap/rockcraft/current/bin/skopeo --insecure-policy copy oci-archive:entrypoint-service-test_latest_amd64v2.rock docker-daemon:entrypoint-service-test:latest
+  rm entrypoint-service-test_latest*.rock  
+  docker images entrypoint-service-test:latest
+  id=$(docker run --rm -d entrypoint-service-test -v)
```

Comment:
why the `-v`? 

What I think we should be doing here is:
`docker run -d entrypoint-service-test` and test that the logs say "foo"

then
`docker run -d entrypoint-service-test bar` and test that the logs say "bar"

p.s. you don't need `--rm` cause the container isn't shutting down on its own and you're deleting later on, below